Makefile

```diff
@@ -0,0 +1,35 @@
+# kernel-style V=1 build verbosity
+ifeq ("$(origin V)", "command line")
+	BUILD_VERBOSE = $(V)
+endif
+
+ifeq ($(BUILD_VERBOSE),1)
+	Q =
+else
+	Q = @
+endif
+
+export GO15VENDOREXPERIMENT:=1
```

Comment:
Was about to comment on that. Let's drop 1.5 and add 1.8 in a followup and remove this env var then